syntax = "proto3";

package xcmg_proto;


message lane_line{
  bool front_wheel_driving_line=1;
  bool blade_operation_line=2;
  bool blade_max_operation_line=3;

  bool set_zero_height = 4;
}
